### DISCUSSION TITLE
How to disable indentation after add new line

### DISCUSSION THREAD

**Discussion by Anifyuli** [LazyVim/LazyVim#2403] [Q&A]
Hello anyone. I get some uncomfortable experience with this problem. If I add new line, I get extra tab on it which is same level with previos tag (in HTML). You can see in my screen recording. What should I do? I still want to use snippets

https://github.com/LazyVim/LazyVim/assets/73734843/81bf727d-3dc3-492c-9a21-8412874a502c

Thanks for your help

**Comment by Anifyuli** [comment]
Solved : Just disabled HTML parser indentation support from `nvim-treesitter`

**Reply by jooo-lee** [reply]
Hey, just wanted to say I had the same problem in HTML when I created a newline using 'o' or 'O' as well. Thanks for sharing your solution! It worked for me.

I put the following code in my `treesitter.lua` file. Hope this helps anyone with a similar problem!
```lua
return {
  "nvim-treesitter/nvim-treesitter",
  opts = {
    indent = {
      -- Fix incorrect indentation for newlines in HTML
      disable = { "html" },
    },
  },
}
```
